pixi.toml

```diff
@@ -117,6 +117,9 @@ pytest = "<=8.0.0"
 pytest-xdist = ">=3.6.1"
 typeguard = ">=4.3.0"
 
+# Dynamixel SDK for LEAP hand
+dynamixel-sdk = "*"
```

Comment:
probably lower bound it still, what i was getting at in the original review is that upper bounding it seemed like a bad default choice